docs/manual/developer_guide.adoc

```diff
@@ -26,23 +26,10 @@ You probably want to use rhel7-branch. Currently we don't support Fedora (unsupp
 ## Build image
 We will create cpio archive `ASCII cpio archive (SVR4 with no CRC)` packed using gzip (`gzip compressed data`).
 
-You can call this script in oscap-anaconda-addon repository. It will create `update.img` image.
+You can use script `create_update_image.sh` in oscap-anaconda-addon repository. It will create `update.img` image.
```

Comment:
Thanks @jan-cerny 

Can you change this to:

```You can use the `create_update_image.sh` script in the oscap-anaconda-addon repository to create the `update.img` image.```

Reply:
@redhatrises Yes, I can!